docs/ops/comparison/IsInf_10.md

```diff
@@ -0,0 +1,63 @@
+# IsInf {#openvino_docs_ops_comparison_IsInf_10}
+
+**Versioned name**: *IsInf-10*
+
+**Category**: *Comparison*
+
+**Short description**: *IsInf* performs element-wise mapping of infinite values to True.
+
+**Detailed description**: *IsInf* performs element-wise mapping of infinite values to true and other values to false. Attributes `detect_negative` and `detect_positive` can be used to control the mapping of negative and positive infinities.
```

Comment:
What if both `detect_negative` and `detect_positive` are `False`?

Reply:
It will return tensor filled with `false`only. I will add comment about it in description